src/constants/index.js

```diff
@@ -0,0 +1,4 @@
+import RDF_PREDICATES from "./rdf_predicates";
+import UPLOAD_TYPES from "./upload_types";
+
+export {RDF_PREDICATES, UPLOAD_TYPES};
```

Comment:
I think we can add a blank line at the end of the file. We've been doing that for the project, so I think we can keep this consistent.